core/tests/karma.conf.ts

```diff
@@ -148,25 +142,37 @@ module.exports = function(config) {
           'node_modules',
         ],
       },
+      devtool: 'inline-source-map',
       module: {
-        rules: [{
-          test: /\.ts$/,
-          use: [
-            'cache-loader',
-            'thread-loader',
-            {
-              loader: 'ts-loader',
-              options: {
-                // this is needed for thread-loader to work correctly
-                happyPackMode: true
+        rules: [
+          {
+            test: /\.ts$/,
+            use: [
+              'cache-loader',
+              'thread-loader',
+              {
+                loader: 'ts-loader',
+                options: {
+                  // this is needed for thread-loader to work correctly
+                  happyPackMode: true
+                }
               }
+            ]
+          },
+          {
+            test: /\.html$/,
+            loader: 'underscore-template-loader'
+          },
+          {
+            test: /\.ts$/,
+            exclude: [ path.resolve(__dirname, "test") ],
+            enforce: 'post',
```

Comment:
What does this mean?

Reply:
This means that `istanbul-instrumenter-loader` is to be run last (to collect coverage).